R/dsl.R

```diff
@@ -132,11 +133,62 @@ add_package_checks <- function(...,
       add_code_step(covr::codecov(quiet = FALSE))
   }
 }
+#' @description
+#' `do_pkgdown_site()` checks if a `id_rsa` env variable is set and builds a pkgdown site.
+#'   A deployment can be avoided by setting `build_only = TRUE`.
+#'
+#' @inheritParams step_build_pkgdown
+#' @inheritParams step_push_deploy
+#' @param build_only Build the pkgdown site but do not deploy it.
+#' @rdname DSL
+#' @export
+#' @importFrom magrittr %>%
+do_pkgdown_site <- function(...,
+  build_only = FALSE,
```

Comment:
`deploy = ci_can_push()` ? Do we want to link to the function from the argument documentation?

Reply:
Added it. After switching to non-R6 notation we get a proper linking to the function in the doc.